pkg/coremodel/alertrule/coremodel.cue

```diff
@@ -0,0 +1,75 @@
+package alertrule
+
+import (
+	"github.com/grafana/thema"
+)
+
+thema.#Lineage
+name: "alertrule"
+seqs: [
+	{
+		schemas: [
+			{// 0.0
+
+				// Unique numeric identifier for the rule. Read-only.
+				id?: int64
+
+				// Unique identifier that can be generated by anyone. string (8-40)
+				uid: string
+
+				folderUid: string
+
+				ruleGroup: string
+
+				title: string
+
+				condition: string
+
+				queries?: [...#AlertQuery]
+
+				// TODO: timestamps
+				// updated
+
+				noDataState: #NoDataState
+
+				execErrState: #ExecutionErrState
```

Comment:
Would i be correct in guessing that these are both things that, ideally, would be stored as metadata that only the system gets to write? That is, it'd be at best meaningless for someone trying to provision an alertrule to include values for these. 

cc @ryantxu

Reply:
No, these are part of provisioning - you get to choose what the rule days if the final evaluation returns "ExecutionError" or "NoData". "Should I alert? Or should I say it's normal?"